### ISSUE TITLE
About SupMAE performance on the cifar-10 dataset

### ISSUE THREAD

**Issue by sybapp** [enyac-group/supmae#1]
Thank you for your work.
```python
def vit_lite_patch4(**kwargs):
    model = VisionTransformer(
        img_size=32, patch_size=4, embed_dim=192, depth=9, num_heads=12, mlp_ratio=2, qkv_bias=True,
        norm_layer=partial(nn.LayerNorm, eps=1e-6), **kwargs)
    return model
```
When I use the above ViT as Encoder, I can get 95.5% Top1 accuracy with MAE (800 epoch), and only 93% Top1 accuracy when using SupMAE(400 epoch). SupMAE should perform better than MAE when using labeled data.

**Comment by Jeff-LiangF** [comment]
@sybapp Thanks for sharing your results! Are you saying that you use your `vit_lite_patch4 ` model to do pre-training on CIFAR-10 and then fine-tuning on CIFAR-10? or pre-training on ImageNet-1K and then fine-tuning on CIFAR-10.

I didn't try any CIFAR-10 experiments yet, but I am curious about the results.
